deepchecks/nlp/utils/text_properties.py

```diff
@@ -25,7 +25,7 @@
 __all__ = ['calculate_default_properties']
 
 
-ONNX_MODELS_STORAGE = pathlib.Path(__file__).absolute().parent / '.onnx-nlp-models'
+MODELS_STORAGE = pathlib.Path(__file__).absolute().parent / '.nlp-models'
```

Comment:
```suggestion
MODELS_STORAGE = pathlib.Path(__file__).absolute().parent / '.deepchecks-nlp-models'
```
Would suggest so it's clear to the user

Reply:
IMO, it is not needed because it will be within `DEEPCHECKS_LIB_PATH/nlp/utils/.nlp-models`

![image](https://user-images.githubusercontent.com/71635444/236855161-41217edb-96b0-4ce9-bc9f-4b1e5c16214c.png)